share/dictionary/radius/dictionary.juniper

```diff
@@ -48,6 +48,7 @@ ATTRIBUTE	Local-Group-Name			46	string
 ATTRIBUTE	Local-Interface				47	string
 ATTRIBUTE	Switching-Filter			48	string
 ATTRIBUTE	VoIP-Vlan				49	string
+ATTRIBUTE	Juniper-CWA-Redirect			50	string
```

Comment:
Please remove the "Juniper-" prefix.  Vendor-specific attributes exist in their own name/number space in master. For example if you were referencing `VoIP-VLAN` in RADIUS, you'd use `Vendor-Specific.Juniper-VoIP-VLAN`

Reply:
Sorry. Done